source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -3,278 +3,528 @@
+   for your mix. They are where ``swupd-client`` will look to figure out if
+   there is a new version available, and the location from which to download
+   the updated content. Think of these as the equivalent of the 
+   `ClearLinux update page`_ used by Clear Linux, but for your derivative mix.
+
+   To learn more about the ``FORMAT`` option, refer to the "Format Version"
+   section at the bottom of this document, and `Format Bumps`_ on the Clear
+   Linux wiki. For now, leave the ``FORMAT`` value alone and do not increment
+   it.
+
+   The mix version and upstream Clear version will come from two state files:
+   :file:`mixversion` and :file:`upstreamversion`, both of which will be
+   created for you when you set up the workspace. They will be created in the
+   directory defined by the ``VERSIONS_PATH``.
+
+#. **Create/locate RPMs for mix.**. (Steps 4 through 6 are necessary only
+   if you want to add your own RPMs to the Mix. If you are working only with
+   Clear bundles, then skip to Step 7.)
+
+   If you are creating RPMs from scratch, you can use ``autospec``, ``mock``,
+   ``rpmbuild``, etc. to build them. If they are not built on Clear,
+   make sure your configuration and toolchain builds them correctly for Clear,
+   or there is no guarantee they will be compatible.
+
+#. **Import RPMs into workspace**. Create a :file:`local-rpms` directory in your
+   workspace (for example :file:`/home/clr/mix/local-rpms`), and copy the RPMs
+   you want into that directory. Next, add the following to your
+   :file:`builder.conf`:
+
+   .. code-block:: bash
+
+      LOCAL_RPM_DIR=/home/clr/mix/local-rpms
+
+   Mixer will look in this directory for RPMs to build a local RPM repo for
+   yum to use.
 
 #. **Create a local RPM repo**. Create an empty directory in your workspace
-   named ``local`` and add the paths in your builder.conf::
+   named :file:`local-yum` and add the path in your :file:`builder.conf`:
 
-    RPMDIR=/home/clr/mix/rpms
-    REPODIR=/home/clr/mix/local
+   .. code-block:: bash
 
-    These variables are automatically read; you simply need to run::
+      LOCAL_REPO_DIR=/home/clr/mix/local-yum
+
+   Once these values are configured, you can generate the yum repo by
+   running the following command:
+
+   .. code-block:: bash
 
       # sudo mixer add-rpms
 
-   After the script exits, you should see your RPMs and a repodata directory in
-   ``/home/clr/mix/local``. If the RPMs are not all in the local directory, check
-   to make sure that they are indeed valid RPM files and not corrupt.
+   After the tool exits, you should see your RPMs and a repodata directory in
+   :file:`/home/clr/mix/local-yum`. If the RPMs are not all in this 
+   :file:`local-yum` directory, check to make sure that they are indeed valid
+   RPM files and not corrupt.
+
+#. **Add/remove/edit/list the bundles in your mix**. The bundles in your mix are
+   specified in the Mix Bundle List. This list is stored as a flat file called
```

Comment:
@kevincwells - is there a reason M, B, and L are capitalized in "Mix Bundle List"?

Reply:
I don't want to get into a grammar argument about what constitutes a "proper noun", but the short version is that I was treating it as one, a) to improve clarity and help people understand that this is a specific, unique thing, and b) to be consistent with the way it is written throughout all of the new command help doc information.

Reply:
I can do both a without the capitalization. Sadly, b means that the help doc must be revised at a later date. As any writer can attest, you cannot fight grammar. I'll make sure to mark it clearly to achieve the desired clarity and specificity.

Reply:
Is there actually a reason to remove these capitalizations, here and throughout the commands and code comments? Just to be completely clear, the capitalization is _not_ grammatically incorrect. It _might_ be against our adopted style guide (do we actually have one?), but that's a different concept altogether.

Reply:
@kevincwells Yes there is. The dictionary is clear: https://en.oxforddictionaries.com/spelling/using-capital-letters
We have had a style guide for months. You can find the capitalization rules here: https://clearlinux.org/documentation/clear-linux/reference/collaboration/documentation/grammar
I will follow both the Oxford grammar and our style guide in my final tweaks.

Reply:
@rcaballeromx 

Thanks for linking me to the style guide -- when I asked if we have one, that was indeed a serious question. I have bookmarked the page and will do my best to adhere to it.

In regards to the style guide and the OED link -- I see nothing at all at either link that disputes this capitalization. The OED link is specifically a list of situations in which "you should always use a capital letter" -- it makes no assertions about when you should not. To my surprise, it makes no mention of the word "proper noun", which is often one of the cases cited. The style guide, however, does state that. "Mix Bundle List" is, in fact, a proper noun.

OED's definition for [proper noun](https://en.oxforddictionaries.com/definition/proper_noun) is quite open, defining it through contrast to a [common nun](https://en.oxforddictionaries.com/definition/common_noun): "A noun denoting a class of objects or a concept as opposed to a particular individual."

[Merriam-Webster](https://www.merriam-webster.com/dictionary/proper%20noun), [Dictionary.com](http://www.dictionary.com/browse/proper-noun), and [Wikipedia](https://en.wikipedia.org/wiki/Proper_noun) make the distinction in both their definition of proper noun and common noun: a "proper noun" is a noun that refers to a unique entity, while a "common noun" refers to a non-unique instance from within a class of entities.

While there are many lists, there is only one Mix Bundle List.

This same definition says that we should, in fact, actually capitalize "Mixer" when we are referring to it as the entity: there are many mixers (many things that mix other things), but there is one Mixer (the tool to build Clear Linux update content). And that would 100% be in keeping with the style guide. Where we _should not_ capitalize "mixer" (and the style guide addresses this when referring to not changing the case of software components) is when it is a literal command, referring to the lowercase-m binary executable.

As such, it would be appropriate to write: "To begin use Mixer, run `mixer init`."